data/projects/enhancing_coala_quickstart.md

```diff
@@ -0,0 +1,53 @@
+[coala-quickstart](https://github.com/coala/coala-quickstart) is a tool that
+helps users to quickly get started with coala. It generates a `.coafile`
+(coala's configuration file) that is tailored to the project. Currently,
+coala-quickstart automatically identifies relevant bears for the project
+based on the languages used and has a clean and simple command-line interface 
+for taking user’s input when required.
+
+However, there’s scope for improvement in the following areas:
+
+- Utilizing information from common configuration files: Some projects have
+  configuration files like `Gruntfile.js`, `Gulpfile.js`, `rakefile`,
+  `package.json`, etc which may contain useful data that's relevant to
+  coala-quickstart. For example, if a project's Gulpfile.js specifies which
+  files are to be linted with csslint, we could use this information to target
+  CSSLintBear at exactly these files.
+- Improving the interface: 
+	- Removing non user-friendly warnings.
+	- Increasing automation.
+	- Validate user input before writing to the generated coafile.
+- Continuous Integration: Like other coala repositories, add Appveyor 
+  (Windows) and Travis (OSX) integartions.
+
+In addition to the above changes the applicants are encouraged to come up with
+their own ideas.
+
+
+#### Milestones
+
+##### PREPARATION/BONDING
+
+* The applicant has identified all the kinds configuration files to be worked
+on in the project.
+* The applicant has figured out all the useful information that could be
+extracted from these files.
+
```

Comment:
Two spaces before `extracted` to make it inline with the above point, also do the same with the other points.